.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules
 package-lock.json
 package.json
 *.pyc
-.env
\ No newline at end of file
+.env
+.idea
```

Comment:
Please consider to use gitignore such files globally instead of lining up personal files. `git config --global core.excludesfile` or `$HOME/.config/git/ignore` will be nice opiton. (You can see detail by `man gitignore`)